docs/content/docs/install/manual_demo/_index.md

```diff
@@ -39,44 +39,269 @@ See the [observability documentation](../../patterns/observability/_index.md) fo
 ### OpenShift
 For details on how to install OSM on OpenShift, refer to the [installation guide](../#openshift)
 
-## Deploy the Bookstore Demo Applications
 
-This demo consists of the following applications:
+## Deploy Applications
 
-- `bookbuyer` application that makes requests to `bookstore` to buy books
-- `bookthief` application that makes requests to `bookstore` to steal books
-- `bookstore` application that receives requests from clients to purchase books and makes requests to `bookwarehouse` to restock books
-- `bookwarehouse` application that receives requests from `bookstore` to restock books
+In this section we will deploy 4 different Pods, and we will apply policies to control the traffic between them.
 
-When we demonstrate traffic splitting using SMI Traffic Split, we will deploy an additional application:
+- `bookbuyer` is an HTTP client makeing requests to `bookstore`. This traffic is **permitted**.
+- `bookthief` is an HTTP client and much like `bookbuyer` also makes HTTP requests to `bookstore`. This traffic should be **blocked**.
+- `bookstore` is a server, which responds to HTTP requests. It is also a client making requests to the `bookwarehouse` service.
+- `bookwarehouse` is a server and should respond only to `bookstore`. Both `bookbuyer` and `bookthief` should be blocked.
 
-- `bookstore-v2` application representing version v2 of the `bookstore` application
 
-The `bookbuyer`, `bookthief`, `bookstore`, and `bookwarehouse` demo applications will be installed in their respective Kubernetes Namespaces. For OSM to work, each application Pod must contain an Envoy proxy as a sidecar container. OSM can automatically inject an Envoy proxy sidecar into application Pods. Use the `osm` CLI to add Kubernetes Namespaces to monitor and inject proxies into using the `osm namespace add` command. Once the Namespace is added, OSM will monitor the Namespace for new Pods and automatically inject the Envoy proxy as a sidecar into each Pod created in the Namespace.
+We are going to craft SMI policies, which will bring us to this final desired
+state of allowed and blocked traffic between pods:
 
-### Create the Bookstore Application Namespaces
+| from  /   to: | bookbuyer | bookthief | bookstore | bookwarehouse |
+|---------------|-----------|-----------|-----------|---------------|
+| bookbuyer     |     \     |     ❌     |     ✔    |       ❌       |
+| bookthief     |     ❌     |     \     |     ❌     |       ❌       |
+| bookstore     |     ❌     |     ❌     |     \     |       ✔      |
+| bookwarehouse |     ❌     |     ❌     |     ❌     |      \        |
+
+
+To show SMI Traffic Split, we will deploy an additional application:
+
+- `bookstore-v2` - a server, which is a newer version of the existing `bookstore` application
```

Comment:
what does "newer" version imply? This line made me wonder about how it's different from v1

Reply:
Good question 🤔...
Let me take another stab at this.

Reply:
I reworded this to:

`bookstore-v2` - this is the same container as the first `bookstore` we deployed, but for this demo we will assume that it is a new version of the app we need to upgrade to.

Reply:
yes, sounds good!